data/interfaces/default/home_stats.html

```diff
@@ -111,6 +111,8 @@
             <div id="stats-thumb-${stat_id}" class="dashboard-stats-flat svg-icon platform-${row0['platform_name']} transparent hidden-xs"></div>
             % elif stat_id == 'most_concurrent':
             <div id="stats-thumb-${stat_id}" class="dashboard-stats-flat svg-icon stats-${stat_id} transparent hidden-xs"></div>
+            % elif stat_id == 'top_libraries':
+            <div id="stats-thumb-${stat_id}" class="dashboard-stats-flat svg-icon library-${row0['type']} hidden-xs"></div>
```

Comment:
Libraries can have a custom thumbnail (Library > Edit) so that should be shown here and fallback to the svg icon if there is no custom thumbnail.

Reply:
On what client is an option to edit the library icon or the background image (I tested the web app and Plex for Windows)?
As for me the only way to include those would be editing the database from plex itself, even though I'm not sure if Plex would even take this setting into account for the different clients.

Reply:
Edit the library in Tautulli, not Plex.

Reply:
With this change we have inconsistency in the appearance.
![Screenshot_1](https://user-images.githubusercontent.com/12448284/110619112-b3b59a00-8197-11eb-817f-4730a35a9a5e.png)
This is because thumbnail in the library statistics only change when a http link is provided not a local ressource. Is this intended?

Reply:
That is clear, but currently the custom thumbnail for a library doesn't work at all. The background does, but the default value `/:/resources/movie.png` never worked at least for me. I just realized this now while working on it. The issue comes from `if (thumb.startsWith('http')) {` which messes up the query resulting in a not found image.
I am sorry, if I am missing a point here just want to make sure it works as intended when I'm already working on it.

Reply:
Yes, it's intended. I ignore the `/:/resources/` icons.